Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM ubuntu:18.04 as builder
+FROM ubuntu:22.04 as builder
```

Comment:
You could also update bullseye to `bookworm` or `bookworm-20230703-slim`

```
FROM debian:bullseye-slim as final
```

Reply:
Ooh, let's do that in a follow-up; I don't want to block this release, and want to think about testing that (but I don't anticipate any problems with it).

Reply:
Well, a simple build test worked, I'll go with it.